crates/transport/src/layers/retry.rs

```diff
@@ -47,9 +62,15 @@ impl RetryBackoffLayer {
             max_rate_limit_retries,
             initial_backoff,
             compute_units_per_second,
+            avg_cost: DEFAULT_AVG_COST,
             policy: RateLimitRetryPolicy,
         }
     }
+
+    /// Set the average cost of a request
```

Comment:
This should mention, that by default this uses 17

we can move the docs from the const above to here